container-host-files/etc/scf/config/scripts/patches/fix_chown_blobstore_packages.sh

```diff
@@ -9,29 +9,27 @@ fi
 
 cd "$PATCH_DIR"
 
-patch --force -p3 <<'PATCH'
-diff --git jobs/blobstore/templates/pre-start.sh.erb jobs/blobstore/templates/pre-start.sh.erb
-index 20b3140..1cf5a58 100644
---- jobs/blobstore/templates/pre-start.sh.erb
-+++ jobs/blobstore/templates/pre-start.sh.erb
-@@ -9,7 +9,7 @@ function setup_blobstore_directories {
+patch --force -p0 <<'PATCH'
+--- pre-start.sh.erb	2019-01-31 10:32:11.714131780 -0800
++++ pre-start.sh.erb	2019-01-31 10:34:34.570302914 -0800
+@@ -9,7 +9,7 @@
    local data_dir=/var/vcap/data/blobstore
    local store_tmp_dir=$store_dir/tmp/uploads
    local data_tmp_dir=$data_dir/tmp/uploads
 -  local nginx_webdav_dir=/var/vcap/packages/nginx_webdav
 +  local packages_dir=/var/vcap/packages
-
+ 
    mkdir -p $run_dir
    mkdir -p $log_dir
-@@ -18,12 +18,12 @@ function setup_blobstore_directories {
-   mkdir -p $data_dir
+@@ -19,12 +19,12 @@
    mkdir -p $data_tmp_dir
-
--  local dirs="$run_dir $log_dir $store_dir $store_tmp_dir $data_dir $data_tmp_dir $nginx_webdav_dir ${nginx_webdav_dir}/.."
+ 
+   chown vcap:vcap $store_dir
+-  local dirs="$run_dir $log_dir $store_tmp_dir $data_dir $data_tmp_dir $nginx_webdav_dir ${nginx_webdav_dir}/.."
 -  local num_needing_chown=$(find $dirs -not -user vcap -or -not -group vcap | wc -l)
-+  local dirs="$run_dir $log_dir $store_dir $store_tmp_dir $data_dir $data_tmp_dir $packages_dir"
++  local dirs="$run_dir $log_dir $store_tmp_dir $data_dir $data_tmp_dir $packages_dir"
 +  local num_needing_chown=$(find -L $dirs -not -user vcap -or -not -group vcap | wc -l)
```

Comment:
If we're no longer using their chown loop (why?), then we shouldn't patch their chown loop :)

Reply:
I have to recheck. IIRC the patch as it was did not apply anymore and I simply made it apply without conflict again.

Reply:
I am not sure why you believe that `we're no longer using their chown loop` ... The only thing I see changed in the patch is that `$store_dir` was removed from the list of directories to handle.

Reply:
Ah, sorry, I got confused reading a patch to a patch.  We _didn't_ do it, upstream did (you're just fixing the context here).  Ignore this then.